client/analytics/components/report-error/index.js

```diff
@@ -0,0 +1,48 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Component } from '@wordpress/element';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import { EmptyContent } from '@woocommerce/components';
+import { getAdminLink } from 'lib/nav-utils';
+
+class ReportError extends Component {
+	render() {
+		const { isError, isEmpty } = this.props;
+		let title, actionLabel, actionURL, actionCallback;
+
+		if ( isError ) {
+			title = __( 'There was an error getting your stats. Please try again.', 'wc-admin' );
+			actionLabel = __( 'Reload', 'wc-admin' );
+			actionCallback = () => {
+				// TODO Add tracking for how often an error is displayed, and the reload action is clicked.
+				window.location.reload();
```

Comment:
If the request returns an error, this will cause a reload. Wouldn't the same error occur and we have an infinite loop?